src/parser/pre2017.py

```diff
@@ -207,10 +260,49 @@ def parse_pre2017(self, case_soup: BeautifulSoup) -> Dict:
                         other_event_rows.append(row)
                     if SECTION == "dispositions":
                         disposition_rows.append(row)
+
                 other_event_rows = other_event_rows[::-1]
                 disposition_rows = disposition_rows[::-1]
+
+                # Process disposition rows into structured format
+                dispositions = []
+                for row in disposition_rows:
+                    if len(row) >= 5:
+                        judicial_officer = ""
+                        if len(row[2]) > 18 and row[2].startswith("(Judicial Officer:"):
```

Comment:
This seems to be some of the core logic for finding the location of the potential disposition related info (some more below where if row[1].lower() == "disposition", etc.). It passed with the unit test and parsed out the field nicely. I am curious to see how it might fair against more/different cases in Hays.